x/ibc/02-client/keeper/keeper.go

```diff
@@ -40,6 +40,10 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("x/%s/%s", ibctypes.ModuleName, types.SubModuleName))
 }
 
+func (k Keeper) GetStoreKey() sdk.StoreKey {
```

Comment:
This shouldn't be public. Breaks object capability model

Reply:
addressed on  #6018